Assets/MixedRealityToolkit/Utilities/MaintainBorderLightWidth.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    /// <summary>
+    /// Utility component to keep the border light width a constant size no mater the object scale.
```

Comment:
Could you please explain why we need this class? I also wonder if it's appropriate to stick in utilities. If it's specific to the button, should it go there?

Reply:
Right now the "_BORDER_LIGHT" feature of the MRTK/Standard shader maintains the border width based on the object's scale. For example if you have a border width of 10 pixels on an object of scale (1,1,1) then you scale that same object to (2,2,2) your border will now be 20 pixels wide. Sometimes developers want to maintain the border width no matter the object scale (in the case of the button front plate as it get's compressed, and a few other instances I've encounterted). Right now the only thing using this behavior is the button, but I wouldn't say this behavior is unique to the button. I'm happy to move it elsewhere if there is a more appropriate location?

Reply:
Could we used this for the Bounding Box (flattened) too?
https://github.com/microsoft/MixedRealityToolkit-Unity/issues/4976

Reply:
I bet we could! We should give it a try.